tests/transforms/test_dynamic_one_shot.py

```diff
@@ -98,45 +98,20 @@ def _():
             return qml.probs(wires=[0])
 
 
-@pytest.mark.parametrize("postselect_mode", ["hw-like", "fill-shots"])
-def test_postselect_mode(postselect_mode):
+def test_postselect_mode():
     """Test that invalid shots are discarded if requested"""
     shots = 100
     dev = qml.device("default.qubit")
 
     @qml.set_shots(shots)
-    @qml.qnode(dev, postselect_mode=postselect_mode)
+    @qml.qnode(dev, postselect_mode="hw-like")
```

Comment:
I would prefer if we didn't hard code `hw-like`, and instead xfailed when postselect_mode is fill-shots.

Reply:
But, non-blocking.